Refresh onfido-php after onfido-openapi-spec update (e92d340) (#53)
onfido/onfido-php: CHANGELOG.md

# Changelog

## v7.0.0 24th June 2024

> - Library has refreshed and automatically generated on [Onfido OpenAPI Spec](https://github.com/onfido/onfido-openapi-spec) (release [v3.0.0](https://github.com/onfido/onfido-openapi-spec/releases/tag/v3.0.0))

Reviewer: ```suggestion
- Library was update to be automatically generated from [Onfido OpenAPI Spec](https://github.com/onfido/onfido-openapi-spec) (release [v3.0.0](https://github.com/onfido/onfido-openapi-spec/releases/tag/v3.0.0))
```

Reviewer 2: What about?
```suggestion
- Library was updated and automatically generated from [Onfido OpenAPI Spec](https://github.com/onfido/onfido-openapi-spec) (release [v3.0.0](https://github.com/onfido/onfido-openapi-spec/releases/tag/v3.0.0))
```
```suggestion
- Library has refreshed and automatically generated on [Onfido OpenAPI Spec](https://github.com/onfido/onfido-openapi-spec) (release [v3.0.0](https://github.com/onfido/onfido-openapi-spec/releases/tag/v3.0.0))
```

> - Integration tests have also been implemented

Reviewer: ```suggestion
- Additional integration test coverage
```

Reviewer 2: Actually integration tests were missing

Reviewer: It's still an addition to the 0 coverage we add 😆
